qsiprep/workflows/dwi/merge.py

```diff
@@ -277,22 +329,26 @@ def init_merge_and_denoise_wf(
         mem_gb=mem_gb,
         omp_nthreads=omp_nthreads,
         source_file=source_file,
+        use_phase=False,  # can't use phase with concatenated data
```

Comment:
Is this true?

Reply:
Now that I think about it, probably not. I do think you should keep it disabled until you (or someone else) can do some testing to make sure it's okay.

Reply:
I personally am not a fan of combining before denoising, and it looks like less than 1% of all recorded qsiprep runs have chosen that route. We can leave it false for now but make an issue for the roadmap. I wonder if the scaling of the phase data needs to incorporate all the concatenated runs?